.github/workflows/nextflow-tests.yml

```diff
@@ -4,12 +4,99 @@ name: 'Nextflow tests'
 on:
   workflow_call:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.sha }}
+  cancel-in-progress: true
+
 # Inspired by https://blog.aspect.dev/github-actions-dynamic-matrix
 jobs:
   discover:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/github-script@v7
+        name: Parse event
+        id: parse-event
+        with:
+          script: |
+            /*
+            We need to determine if the incoming event was from a pull_request
+            or an issue_comment
+            */
+            let apply_fixes = context.eventName == 'issue_comment'
+
+            let pr = {}
+
+            let abort_early = false
+
+            if (apply_fixes) {
+              let bot_id = 41898282
```

Comment:
Is this a randomly chosen number? I wonder if there's ever a possibility of the ID matching an actual user at some point

Reply:
No, it's the user ID of the GitHub actions bot: https://api.github.com/users/github-actions%5Bbot%5D, so no other use should ever match it.

Reply:
At some point I had a much fuller comment about this, but I stripped it at some point for some reason:

https://github.com/uclahs-cds/tool-Nextflow-action/blob/035c9423f0de304c609dbd5e8e0b0bdab445a715/.github/workflows/nextflow-tests.yml#L123-L130

Reply:
Got it, makes sense

Reply:
I suggest leaving a comment like `// Actions bot ID - see https://api.github.com/users/github-actions%5Bbot%5D`

Reply:
Done with 43877d5